_developer/templates/developer/template.html

```diff
@@ -1,9 +1,16 @@
-<!--Do Not Remove any parts of this document unless you expect to not use out of the box behaviors-->
+<!--Do Not Remove any parts of this document unless you expect to not use out of the box behaviours-->
```

Comment:
change to "behaviors" is this Canada? ;)

Reply:
Fine. Fixed. LOL